siliconcompiler/core.py

```diff
@@ -4156,7 +4156,7 @@ def run(self):
                            fatal=True)
             pre_remote_steplist = {
                 'steplist': self.get('option', 'steplist'),
-                'set': self.get('option', 'steplist', field='set'),
+                'set': self.schema._is_set(self.schema._search('option', 'steplist')),
```

Comment:
I think this should be: `self.schema._is_set('option', 'steplist'),`

Reply:
nevermind, I see what it's doing

Reply:
`Schema._is_set` is a private method which accepts a config dictionary, not a keypath. `Schema._search` returns the config dictionary for a given keypath.